source/tutorials/multi-boot/multi-boot.rst

```diff
@@ -0,0 +1,181 @@
+.. _multi-boot:
+
+Multi-boot |CL-ATTR| and Windows\* 10 OS
+########################################
+
+This guide shows how to install |CL-ATTR| adjacent to an existing Windows\*
+10 :abbr:`OS (operating system)`. To add |CL| to an existing Windows installation, there are two possible methods:
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
+In this tutorial you'll install |CL| :ref:`bare-metal-install-desktop` as
+an additional partition.
+
+Method 1: Shrink the Windows partition to make space for |CL|
+*************************************************************
+
+#. Boot up the Windows 10 OS.
+
+#. Launch the :file:`Disk Management` utility found under
+   :guilabel:`Create and format hard disk partitions`.
+
+#. Right-click the primary Windows partition and select “Shrink Volume...”.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-01.png
+      :width: 75%
+      :alt: Disk Management > Shrink Volume
+
+      Figure 1: Disk Management > Shrink Volume.
+
+#. Shrink the size of the partition by at least the following amount:
+
+   * For :ref:`bare-metal-install-desktop`, allow at least 21GB.
+
+   * For :ref:`bare-metal-install-server`, allow at least 4GB.
+
+#. We shrink C from about 91000 to 70000, as shown in Figure 2.
```

Comment:
The wordings and the figure do not match up.
If you shrink from 91000 to 70000, then the last field "Total size after shrink in MB" should be "21,000", not "42,907" as shown in the figure.

Reply:
Sorry about that. Result of doctored image.